### PR TITLE
Adding utility to get K8 version

### PR DESCRIPTION
**Pull request by sarveshkaushal**
My local workspace got corrupted. So created new PR.

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/version/Version.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  public Version() {
+    this(Configuration.getDefaultApiClient());
+  }
+
+  public Version(ApiClient apiClient) {
+    this.versionApi = new VersionApi(apiClient);
+  }
+
+  public VersionInfo getVersion() throws ApiException, IOException {
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/util/version/Version.java:36]: Does this actually ever throw IOException?

File: util/src/test/java/io/kubernetes/client/util/version/VersionTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  private static final int PORT = 8091;
+  @Rule public WireMockRule wireMockRule = new WireMockRule(PORT);
+
+  @Before
+  public void setup() throws IOException {
+    client = new ClientBuilder().setBasePath("http://localhost:" + PORT).build();
+  }
+
+  @Test
+  public void testUrl() throws InterruptedException, IOException, ApiException {
```
Comment (brendandburns) [inline util/src/test/java/io/kubernetes/client/util/version/VersionTest.java:45]: Add a test for a non-200 return code.

**Comment by brendandburns** [comment]
Looks good, two more comments.

File: util/src/main/java/io/kubernetes/client/util/version/Version.java (hunk 1 above)
Comment (sarveshkaushal) [reply util/src/main/java/io/kubernetes/client/util/version/Version.java:36]: It actually does but it's not able to reach K8 cluster for some reason. But I will need to handle this.

Comment (sarveshkaushal) [reply util/src/main/java/io/kubernetes/client/util/version/Version.java:36]: Sorry i meant, the code to deserialize response will throw IOException.

File: util/src/test/java/io/kubernetes/client/util/version/VersionTest.java (hunk 2 above)
Comment (sarveshkaushal) [reply util/src/test/java/io/kubernetes/client/util/version/VersionTest.java:46]: Done

File: util/src/main/java/io/kubernetes/client/util/version/Version.java (hunk 1 above)
Comment (brendandburns) [reply util/src/main/java/io/kubernetes/client/util/version/Version.java:36]: ah, ok.

**Pull request merged**